init.js

```diff
@@ -2,9 +2,61 @@
 
 /**
  * Archetype `init` configuration.
+ *
+ * See: https://github.com/FormidableLabs/builder-init/blob/master/README.md#archetype-data
+ * for structuring this configuration file.
  */
-// TODO: REMOVE AND IMPLEMENT PROMPTS
-// https://github.com/FormidableLabs/builder-init/issues/3
-/*eslint-disable no-console*/
-console.log("TODO HERE init.js");
-/*eslint-enable no-console*/
+module.exports = {
+  // Prompts are Inquirer question objects.
+  // https://github.com/SBoudrias/Inquirer.js#question
+  //
+  // `builder-init` accepts:
+  // - an array of question objects (with a `name` property)
+  // - an object of question objects (keyed by `name`)
+  //
+  prompts: {
+    licenseDate: {
+      message: "License date",
+      default: (new Date()).getFullYear().toString()
+    },
+    licenseOrg: {
+      message: "License organization (e.g., you or your company)",
+      validate: function (val) {
+        return !!val.trim() || "Must enter a license organization";
+      }
+    },
+    // See: https://github.com/npm/validate-npm-package-name
+    packageName: {
+      message: "Package / GitHub project name (e.g., 'whiz-bang-component')",
```

Comment:
@zachhale -- No, because this _creates_ the project directory. It will output:

```
{{ packageName }}/
  package.json  // (Which "name": "{{ packageName }}")
```